src/components/molecules/MenuPopoverContent/MenuPopoverContent.scss

```diff
@@ -0,0 +1,26 @@
+@import "scss/constants.scss";
+
+$menu-popover-width: 180px;
+
+// @debt hack-fix to update popover sizes
+#menu-popover {
```

Comment:
What was the issue that requires `#id` type of selector?

Reply:
i assume that's because we're using some sort of an overlay there that has to have `id` in order for styles to apply
![image](https://user-images.githubusercontent.com/56254806/130601264-73c0fdcd-ab5c-4620-ae02-99e4384b68eb.png)